style.css

```diff
@@ -0,0 +1,79 @@
+* {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+}
+
+@media screen and (max-width: 767px) {
+  html {
```

Comment:
- [ ] Since in this milestone we are working on mobile versions of your portfolio, it would be better for you to write your code without using media queries so that in future milestones when you will be working on desktop versions, you will introduce media queries.